.vscode/launch.json

```diff
@@ -2,7 +2,7 @@
     "version": "0.2.0",
     "configurations": [
         {
-            "name": "Integrated Terminal/Console",
+            "name": "Azure CLI Debug (Integrated Console)",
```

Comment:
What is the purpose of the changes to this file?

Reply:
To add more clarity to the name of the debug option. `Integrated Terminal/Console` itself might be a bit confusing.

Reply:
I meant all of the changes in the file. :)  Is it just VSCode convenience stuff?

Reply:
Yes that's correct!